spec/requests/v1/api_spec.rb

```diff
@@ -208,7 +208,7 @@
     end
 
     describe 'POST /api/v1/tasks' do
-      let(:params) { {task: { name: 'New Task', url: "http://wwww.gsa.gov/", task_items_attributes: [{name: "Item 1", external_id: "abc"}]}} }
+      let(:params) { {task: { name: 'New Task', url: "http://wwww.gsa.gov/", task_items_attributes: [{name: "Item 1", external_id: "abc", url: "http://www.gsa.gov/"}]}} }
```

Comment:
Line is too long. [170/100]<br>Space inside { missing.<br>Space inside } missing.